ocaml/xapi/sparse_dd_wrapper.ml

```diff
@@ -70,8 +70,11 @@ let dd_internal progress_cb base prezeroed infile outfile size =
 					"-machine";
 					"-src"; infile;
 					"-dest"; outfile;
-					"-size"; Int64.to_string size
-				] @ (if prezeroed then [ "-prezeroed" ] else []
+					"-size"; Int64.to_string size;
+					"-good-ciphersuites"; (match !Xapi_globs.ciphersuites_good_outbound with None -> "!EXPORT:RSA+AES128-SHA256" | Some s -> s) ;
```

Comment:
We must not have any hardcoded default values here.
This should be handled as in `ocaml/xapi/xapi_sync.ml`, with
```
(match !Xapi_globs.ciphersuites_good_outbound with
    | Some s -> s
    | None -> raise (Api_errors.Server_error (Api_errors.internal_error,["Vdi_copy found no good ciphersuites in Xapi_globs."]))
);
```